front/lib/auth.ts

```diff
@@ -109,13 +112,16 @@ export class Authenticator {
 
     let role = "none" as RoleType;
 
-    if (workspace && keyWorkspace) {
-      if (keyWorkspace.id === workspace.id) {
+    if (workspace) {
+      if (keyWorkspace!.id === workspace.id) {
```

Comment:
we should always have a workspace for the Key. If not that's fine to crash and burn